apps/src/sites/code.org/pages/views/swiper_page_videos.js

```diff
@@ -24,7 +24,6 @@ const swiperParams = {
     clickable: true,
   },
   spaceBetween: 24,
-  observeSlideChildren: true,
```

Comment:
Removing this unused param on the code.org/videos page JS file. This was a leftover from when I was trying to make the lightbox video player work, bit it's not doing anything.